### PR TITLE
Fix viewport not updating when denoiser enabled

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-3266

PURPOSE: Currently when denoiser and/or tonemapper is enabled in viewport render it is not updated correctly after first render

EFFECT FOR THE USER: Viewport render working as expected

NOTE FOR REVIEWERS: Also updated some debug dump functions in this PR

### CODE REVIEW

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 1]
```diff
@@ -1643,6 +1645,8 @@ std::vector<float> FireRenderContext::GetDenoisedData(bool& result)
 {	
 #ifdef _DEBUG
 #ifdef DUMP_FRAMEBUFF
+	const std::string pathToFile = "C://debug//fb//";
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:1648]: please remove

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 2]
```diff
@@ -1652,22 +1656,22 @@ std::vector<float> FireRenderContext::GetDenoisedData(bool& result)
 		m_pixelBuffers[RPR_AOV_COLOR].debugDump(m_pixelBuffers[
 			RPR_AOV_COLOR].height(),
 				m_pixelBuffers[RPR_AOV_COLOR].width(),
-				std::string("RPR_AOV_COLOR"));
+				std::string("RPR_AOV_COLOR"), pathToFile);
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:1659]: Pls remove "pathToFile"if we should not have it here in release mode

File: FireRender.Maya.Src/Context/FireRenderContext.cpp [hunk 3]
```diff
@@ -1652,22 +1656,22 @@ std::vector<float> FireRenderContext::GetDenoisedData(bool& result)
...
 
 		m_pixelBuffers[RPR_AOV_SHADING_NORMAL].debugDump(m_pixelBuffers[
 			RPR_AOV_SHADING_NORMAL].height(),
 				m_pixelBuffers[RPR_AOV_SHADING_NORMAL].width(),
-				std::string("RPR_AOV_SHADING_NORMAL"));
+				std::string("RPR_AOV_SHADING_NORMAL"), pathToFile);
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/FireRenderContext.cpp:1664]: same

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]

**Review by Vsevolod1983 — APPROVED** [review]

**Comment by DavidSpeicher** [comment]
QA approved

**Pull request merged**